bigbluebutton-html5/imports/api/2.0/cursor/server/modifiers/initializeCursor.js

```diff
@@ -0,0 +1,8 @@
+import Cursor from './../../';
+import updateCursor from './updateCursor';
+
+export default function initializeCursor(meetingId) {
+  check(meetingId, String);
+
+  return updateCursor(meetingId, 0, 0);
```

Comment:
A cursor position of (0,0) is a valid position and shouldn't be the default value. A better default is (-1,-1) because that can just be set to "visible=false" or something when rendered.